src/graphql/mutations/__tests__/__snapshots__/UpdateUser.js.snap

```diff
@@ -20,7 +20,7 @@ Object {
 
 exports[`UpdateUser should clear avatarData field for non openpeeps avatar 1`] = `
```

Comment:
nit: since we do not clear `avatarData` when openpeeps is not chosen, I think we can update the test name accordongly.